client/my-sites/site-settings/disconnect-site/disconnect.jsx

```diff
@@ -0,0 +1,82 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { isFreeJetpackPlan } from 'lib/products-values';
+import SelectDropdown from 'components/select-dropdown';
+
+class Disconnect extends Component {
+	state = {
+		reasonSelected: 'tooHard',
+		compactButtons: false,
+		renderFull: false,
+	};
+
+	renderFull() {
+		// placeholder
+		return (
+			<div>
+				{ ' follow-up QA' }
+			</div>
+		);
+	}
+
+	logReason = option => {
+		this.setState( {
+			reasonSelected: option.value,
+			renderFull: true,
+		} );
+	};
+
+	getOptions() {
+		const { site } = this.props;
+
+		const options = [
+			{ value: 'tooHard', label: 'It was too hard to configure Jetpack' },
+			{ value: 'didNotInclude', label: 'This plan didn’t include what I needed' },
+		];
+
+		if ( ! isFreeJetpackPlan( site.plan ) ) {
+			options.push( { value: 'onlyNeedFree', label: 'This plan is too expensive' } );
+		}
+		return options;
+	}
+
+	render() {
+		const { translate, siteSlug } = this.props;
+		const { reasonSelected, compactButtons, renderFull } = this.state;
+
+		const textShareWhy =
+			translate( ' Would you mind sharing why you want to' + 'disconnect ' ) +
+			`${ siteSlug }` +
+			translate( ' from WordPress.com ' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Updates from WordPress.com' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/site-settings-disconnect-site-survey-separate-reconnection)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).